src/main/java/com/sprint/mission/discodeit/service/basic/BasicBinaryContentService.java

```diff
@@ -0,0 +1,41 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.binarycontent.BinaryContentCreateDTO;
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.repository.BinaryContentRepository;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor //final 혹은 @NotNull이 붙은 필드의 생성자를 자동 생성하는 롬복 어노테이션
+public class BasicBinaryContentService implements BinaryContentService {
+    final private BinaryContentRepository binaryContentRepository;
```

Comment:
private 다음에 final이 오는게 맞을 것 같아요.  

```suggestion
    private final BinaryContentRepository binaryContentRepository;
``` 

참고 : [Google Java Coding Convention #4.8.7-modifiers](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)
> Class and member modifiers, when present, appear in the order recommended by the Java Language Specification:
```
    public protected **private** abstract default static **final** transient volatile synchronized native strictfp
```